src/yb/tserver/tablet_service.cc

```diff
@@ -2479,8 +2479,10 @@ void ConsensusServiceImpl::GetConsensusState(const consensus::GetConsensusStateR
     return;
   }
   LeaderLeaseStatus leader_lease_status;
-  *resp->mutable_cstate() = scope->ConsensusState(req->type(), &leader_lease_status);
+  bool leader_no_op_committed_;
+  *resp->mutable_cstate() = scope->ConsensusState(req->type(), &leader_lease_status, &leader_no_op_committed_);
```

Comment:
This line seems too long, according to yb coding style, each line should be shorter than 100. Refer to
[Coding style](https://docs.yugabyte.com/preview/contribute/core-database/coding-style/#line-length), it could be modified to
```
*resp->mutable_cstate() = scope->ConsensusState(
    req->type(), &leader_lease_status, &leader_no_op_committed_);
```
It's wrapped and the second line start with 4 spaces